package.json

```diff
@@ -35,7 +37,7 @@
   },
   "homepage": "https://github.com/0xPolygonHermez/zkevm-commonjs#readme",
   "devDependencies": {
-    "@0xpolygonhermez/zkevm-contracts": "github:0xPolygonHermez/zkevm-contracts#v3.0.0-rc.1-fork.6",
+    "@0xpolygonhermez/zkevm-contracts": "github:0xPolygonHermez/zkevm-contracts#feature/l1-info-tree",
```

Comment:
Do we have tag?

Reply:
tests will fail since we need still to migrate to new bridge LxLy